docs/common_tasks.rst

```diff
@@ -120,6 +120,25 @@ To add a new repository, the following steps are needed:
      > sudo /etc/init.d/supervisord restart
 
 
+Add a new full access user
+--------------------------
+
+To add a new user with full staff access to Treeherder's interface, open a python shell in vagrant
+
+  .. code-block:: bash
+
+      (venv)vagrant@local:~/treeherder$ python manage.py shell
+
+Then type:
+
+  .. code-block:: python
+
+      >>> from django.contrib.auth.models import User
+      >>> u = User.objects.get(email="your@email.com")
+      >>> u.is_staff = True
+      >>> u.save()
```

Comment:
@tojonmz yeah the createsuperuser command is the one to create a new superuser from scratch. The python snippet is just to promote an existing user to superuser. Feel free to wrap it in a new 'promotesuperuser' django command if you like it.